reactive/ovn_chassis_charm_handlers.py

```diff
@@ -88,6 +88,10 @@ def configure_ovs():
             charm.optional_interfaces((ovsdb,),
                                       'nova-compute.connected',
                                       'amqp.connected'))
+        if not ovsdb.db_sb_connection_strs:
+            ch_core.hookenv.log('No OVNSDB connection details available',
+                                level=ch_core.hookenv.INFO)
+            return
```

Comment:
The early return here would prevent the charm from performing other configuration steps, I wonder if it would make more sense to put the check around this section and log a warning instead of executing the command:
https://github.com/openstack-charmers/charm-layer-ovn/blob/8ef21ec3675c33a0c72a569f736b83f93fdee86c/lib/charms/ovn_charm.py#L1007-L1019